src/it/scala/org/ergoplatform/it/TestFourNodesSuite.scala

```diff
@@ -16,7 +16,10 @@ class TestFourNodesSuite extends FreeSpec with BeforeAndAfterAll with ScorexLogg
     """.stripMargin
   )
 
-  private val docker = Docker(getClass)
+  private val docker: Docker = {
+    Docker.cleanupResourcesIfNeeded()
```

Comment:
I thing it worth moving this call to `docker.close` and move the `cleanupResourcesIfNeeded method`  to `Docker` class, not an object

Reply:
Make sense. I'll move the `cleanupResourcesIfNeeded ` call to `startNodes`  then

Reply:
It'll be too late. We create docker network in `Docker.innerNetwork` on instantiation.

Reply:
oh, indeed! Then I'll but it to initialization section just after client creation